wp/lib/bap_wp/bap_wp.opam

```diff
@@ -19,8 +19,8 @@ bug-reports: "https://github.com/draperlaboratory/cbat_tools"
 dev-repo: "git+https://github.com/draperlaboratory/cbat_tools"
 depends: [
   "bap" {>= "2.1.0"}
-  "core_kernel" {>= "v0.14.0"}
-  "z3" {>= "4.8.9"}
-  "ounit2" {>= "2.2.3"}
-  "re" {>= "1.9.0"}
+  "core" {>= "0.12.4"}
+  "z3" {>= "4.8.6"}
+  "ounit"
```

Comment:
Any reason for changing ounit2 to ounit?

Reply:
This was arbitrary on my end. I'll change it to `ounit2`.

Reply:
This is generally not good. Depending on `core` is tedious. Basically any change to the `opam` file is somewhat sus.

Are any of these changes necessary?

Reply:
They were necessary at one point to get `make` working on my end, but now seem to be unnecessary. I'll change this back.